src/casp/services/api/schemas.py

```diff
@@ -16,3 +16,22 @@ class MatchInfo(BaseModel):
 class QueryCell(BaseModel):
     query_cell_id: str = Field(example="99543")
     matches: t.List[MatchInfo]
+
+
+class CASModel(BaseModel):
+    system_name: str = Field(example="incremental-pca-001")
```

Comment:
Why `system_name` instead of something like `model_name`?

Reply:
Because it is already part of `CASModel` which means that this is a system name of the model.